### PR TITLE
AM200, EM10 changes

### PR DESCRIPTION
**Pull request by MichaelDvP**
I've reduced to the changes that are confirmed working. 
The EM10 is now mapped to gateway, I don't see why interface fits better than gateway.
Valves and other entities from AM200 can be added later.
wwFlowTempOffset max is set to 45.

### CODE REVIEW

File: src/web/WebSettingsService.cpp [hunk 1]
```diff
@@ -274,6 +274,7 @@ void WebSettingsService::onUpdate() {
 
 void WebSettingsService::begin() {
     _fsPersistence.readFromFS();
+    WebSettings::reset_flags();
```
Comment (proddy) [inline src/web/WebSettingsService.cpp:277]: good catch!

**Comment by proddy** [comment]
shall we add a line to the changelog and bump up the version number?

**Comment by MichaelDvP** [comment]
I think we should wait with version bump and changelog until the AM200 is completed. Or, maybe  add " Start integrating AM200" now?

**Comment by proddy** [comment]
let's wait then

**Pull request merged**